.github/fixtures/test-ignore-tags/cliff.toml

```diff
@@ -19,15 +18,19 @@ body = """
     {% endfor %}
 {% endfor %}\n
 """
-# template for the changelog footer
-footer = """
+# A Tera template to be rendered as the changelog's footer (see https://keats.github.io/tera/docs/#introduction).
+footer_template = """
 <!-- generated by git-cliff -->
 """
-# remove the leading and trailing whitespace from the templates
-trim = true
+# Whether to remove leading and trailing whitespaces from all lines of the changelog's body.
+trim_body_whitespace = true
 
-[git]
-# regex for skipping tags
-skip_tags = "v0.1.0-beta.1"
-# regex for ignoring tags
-ignore_tags = "v.*-beta.*"
+[release]
+# Regex to select git tags that do not represent proper releases. Takes precedence over `release.tags_pattern`.
+# Changes belonging to these releases will be included in the next non-skipped release.
+# Example: "rc"
+skip_tags_pattern = "v.*-beta.*"
+
+[commit]
+# Regex to select git tags that should be excluded from the changelog.
+exclude_tags_pattern = "v0.1.0-beta.1"
```

Comment:
Should we pick one between `exclude_` and `skip_`?

Reply:
`exclude_` and `skip_` have different meanings. Changes from excluded tags are completely removed from the changelog, whereas changes from skipped tags are contained in the next release thereafter. It certainly is not ideal naming but I could not find a better option.